warehouse/query-core/src/main/java/datawave/query/iterator/builder/IndexFilterIteratorBuilder.java

```diff
@@ -76,8 +76,8 @@ public void setRangeAndFunction(LiteralRange range, Filter filter) {
                                 .withTimeFilter(timeFilter).withDatatypeFilter(datatypeFilter).negated(false)
                                 .withScanThreshold(ivaratorCacheScanPersistThreshold).withScanTimeout(ivaratorCacheScanTimeout)
                                 .withHdfsBackedSetBufferSize(ivaratorCacheBufferSize).withMaxRangeSplit(maxRangeSplit).withMaxOpenFiles(ivaratorMaxOpenFiles)
-                                .withFileSystem(hdfsFileSystem).withUniqueDir(new Path(hdfsCacheURI)).withQueryLock(queryLock).allowDirResuse(true)
-                                .withReturnKeyType(PartialKey.ROW_COLFAM_COLQUAL_COLVIS_TIME).withSortedUUIDs(sortedUIDs)
+                                .withMaxSize(ivaratorMaxSize).withFileSystem(hdfsFileSystem).withUniqueDir(new Path(hdfsCacheURI)).withQueryLock(queryLock)
+                                .allowDirResuse(true).withReturnKeyType(PartialKey.ROW_COLFAM_COLQUAL_COLVIS_TIME).withSortedUUIDs(sortedUIDs)
                                 .withCompositeMetadata(compositeMetadata).withCompositeSeekThreshold(compositeSeekThreshold).withTypeMetadata(typeMetadata)
```

Comment:
Not related to the changes in this PR, but perhaps we could put each piece of the builder on it's own line and surround with // @formatter: off ?

Reply:
ok, reasonable